arviz/kde_utils.py

```diff
@@ -495,6 +504,84 @@ def kde(x, circular=False, **kwargs):
     grid : Gridded numpy array for the x values.
     pdf : Numpy array for the density estimates.
     bw: optional, the estimated bandwidth.
+    
```

Comment:
Probably a better place for these examples is `plot_kde`. Also a circular example could be added to the example gallery

Reply:
Could it be tackled in another PR? 
I've updated `plot_kde()` and the other functions to use `kde()` but they are not passing extra arguments to `kde()` yet. When I tried to do it I saw it required to modify several things related to how the call is done and I preferred to keep this PR more focus on the KDE itself.

Reply:
yes, but open an issue as a reminder.

Reply:
You can link to plot_kde with rst to make sure people reading this docstring know to go there for examples: https://arviz-devs.github.io/arviz/developer_guide.html#hyperlinks They should both probably refer one another in the See also section.